2-projects/1-todo-list/style.css

```diff
@@ -10,3 +10,15 @@ body {
     margin-top: 150px;
     width: 90%;
   }
+
+  .li-sec {
```

Comment:
li-sec and list-sec are basically the same things. To avoid confusion, it would be beneficial to use a more specific naming convention, such as "todo-list-sec," to differentiate it from other similarly named sections. This will be especially important for other engineers who will be working with the codebase, as clear and specific naming will reduce the amount of time they need to spend figuring out what each section does 🤓